src/model/DiseaseInvolvement.tsx

```diff
@@ -0,0 +1,125 @@
+import React, { Fragment } from 'react';
+import { v1 } from 'uuid';
+import { ExternalLink } from 'franklin-sites';
+import CommentType from './types/commentType';
+import { EvidenceType } from './types/modelTypes';
+import UniProtEvidenceTag from '../components/UniProtEvidenceTag';
+import { DatabaseToDatabaseInfo } from './types/databaseTypes';
+import databaseToDatabaseInfoJson from '../data/databaseToDatabaseInfo.json';
+import { DatabaseCrossReference } from './XRef';
+
+const databaseToDatabaseInfo: DatabaseToDatabaseInfo = databaseToDatabaseInfoJson;
+
+type DiseaseType = {
+  diseaseId?: string;
+  diseaseAccession?: string;
+  acronym?: string;
+  description?: string;
+  reference?: DatabaseCrossReference;
+  evidences?: EvidenceType[];
+};
+
+export type DiseaseCommentType = {
+  commentType?: CommentType.DISEASE;
+  disease?: DiseaseType;
+  note?: { texts?: { value?: string }[] };
+};
+
+type DiseaseInvolvementEntryProps = { comment: DiseaseCommentType };
+
+type DiseaseInvolvementProps = {
+  data: {
+    comments?: DiseaseCommentType[];
+  };
+};
+
+export const DiseaseInvolvementEntry: React.FC<
+  DiseaseInvolvementEntryProps
+> = ({ comment }) => {
+  const { disease, note } = comment;
+  if (!disease) {
+    return null;
+  }
+  const {
+    diseaseId,
+    acronym = '',
+    evidences,
+    description,
+    reference,
+  } = disease;
+  if (!diseaseId) {
+    return null;
+  }
+  const evidenceNodes =
+    evidences &&
+    evidences.map(evidence => (
+      <UniProtEvidenceTag evidence={evidence} key={evidence.id} />
+    ));
+
+  let notesNode;
+  if (note) {
+    const { texts } = note;
+    if (texts) {
+      notesNode = texts.map(text => text.value).join(' ');
```

Comment:
I don't know if it's best to join them or to have them enclosed within <p> tags?

Reply:
Is the blank space template string back ticks ` `` `? If so, could you expand on what you're thinking.

Reply:
`texts` is an array of strings, which I think you are converting into a long string, with each array item separated by a space? If so I was wondering if it would be better to have each array item as a `<p/>` instead (maybe not!)

Reply:
I've made these unordered list with no bullets